test/Microsoft.AspNetCore.Mvc.Core.Test/ModelBinding/ParameterBinderTest.cs

```diff
@@ -381,7 +381,7 @@ public static TheoryData BindModelAsyncData
             ModelMetadata metadata)
         {
             // Arrange
-            var expectedKey = metadata.Name ?? string.Empty;
+            var expectedKey = string.Empty;
```

Comment:
Nice. I thought some of the tests I just added might need to be fixed. We were previously missing a few tests around the various `ModelMetadata` naming cases.